build/ci/setup-environment.yml

```diff
@@ -22,75 +20,29 @@ steps:
     condition: ne('$(dotnetVersion)', '')
 
   - task: UseDotNet@2
-    displayName: 'Use dotnet 10.0.100-preview.3.25201.16'
+    displayName: 'Use dotnet $(dotnetNextVersion)'
     inputs:
-      version: 10.0.100-preview.3.25201.16
+      version: $(dotnetNextVersion)
       performMultiLevelLookup: true
       includePreviewVersions: true
```

Comment:
Should we make this use the latest .NET 10 preview?

I'm not sure if I want to revisit this each month and bump to the next one?

The only concern is if we think a new release would break.